### ISSUE TITLE
create a shared layout for all pages

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/www.greenwoodjs.dev#32]
## Summary

We will have a common layout for all pages that will need to include the theme CSS, footer, header, etc.

> Depends on #23 , #26, #27

## Summary

1. [ ] Create a _src/templates/app.html_
1. [ ] Should incorporate the _theme.css_ file for all design system branding
1. [ ] Should include the common header component
1. [ ] Should have a `<page-outlet></page-outlet>` placeholder for all pages content
1. [ ] Should include the common footer component
1. [ ] Should have all `<meta>` tags from the [current website](https://github.com/ProjectEvergreen/greenwood/blob/master/www/templates/app.html)

## Open Items
1. [ ] Depends on #23 , #26, and #27
1. [x] ~~(nice to have) Grid lines / graph paper aesthetic - https://github.com/ProjectEvergreen/www.greenwoodjs.dev/issues/32#issuecomment-2118221206~~ - nvm, this is only on the #30

**Comment by thescientist13** [comment]
@Auhseh 
Had a question coming out of #44 where-in I wanted make sure as part of this ticket we account for the "grid lines" / graph paper aesthetic in our design, because I'm not actually sure if that's just part of Figma, or if its actually meant to be part of our designs?
![Screenshot 2024-05-17 at 3 03 24 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/6cf7872b-0eda-4a2e-9d53-c638695cdea9)

If it is part of our design, it does feel / look similar to the Vercel and Next home pages, so not sure if that was intentional or not.
![Screenshot 2024-05-17 at 3 06 17 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/03068f7f-204e-4178-8d54-4491499ababa)
![Screenshot 2024-05-17 at 3 06 26 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/5d0b1683-514c-43ce-81e4-e9dbfa38b3e5)

I don't feel too strongly about it, but in particular if it's already part of someone elee's design, I would prefer not to use it.

**Comment by Auhseh** [comment]
Just seeing this…. it’s part of the design 
It’s actually closer to Cal than Vercel/Next. (Adds a little oomph to the design)

But I totally understand you 🥲

**Comment by thescientist13** [comment]
No worries, we can evaluate at the time of working on this particular task how we feel about it based on if it's much effort or not to achieve.
